src/jsonrpc/miner_jsonrpc_snapshot.erl

```diff
@@ -14,13 +14,29 @@
 handle_rpc(<<"snapshot_list">>, []) ->
     Chain = blockchain_worker:blockchain(),
     case blockchain:find_last_snapshots(Chain, 5) of
-        undefined -> ?jsonrpc_error(no_snapshots_found);
+        undefined ->
+            ?jsonrpc_error(no_snapshots_found);
         Snapshots ->
-            [ #{ height => Height,
-                 hash => ?BIN_TO_B64(Hash),
-                 hash_hex => blockchain_utils:bin_to_hex(Hash),
-                 have_snapshot => have_snapshot(Hash, Chain) } ||
-              {Height, _, Hash} <- Snapshots ]
+            [
+                #{
+                    height => Height,
+                    hash => ?BIN_TO_B64(Hash),
+                    hash_hex => blockchain_utils:bin_to_hex(Hash),
+                    have_snapshot => have_snapshot(Hash, Chain)
+                }
+             || {Height, _, Hash} <- Snapshots
+            ]
+    end;
+handle_rpc(<<"snapshot_info">>, [#{<<"path">> := Path}]) ->
+    case file:read_file(Filename) of
```

Comment:
where does this filename come from? it doesn't seem to be bound in the parameters?